### PR TITLE
Using gimme to provide Go versions for the tests

### LINKED ISSUE #39: Use `gimme` to test zypper-docker with different golang versions
Currently we have a Docker image for each image that we want to test. This can be a bit awkward in the future. My proposal is to have just one Dockerfile, that uses [gimme](https://github.com/travis-ci/gimme). This is what Travis does to handle Go versions, and I think that is a good idea.

@flavio what do you think?

### PR DESCRIPTION
**Pull request by jloehel**
Fixes #39

Signed-off-by: Jürgen Löhel <jloehel@suse.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,22 +1,18 @@
...
-	@echo Running climate inside of tip
-	docker run --rm -v `pwd`:/go/src/github.com/SUSE/zypper-docker zypper-docker-testing-tip climate -open=false -threshold=80.0 -errcheck -vet -fmt .
-
-test :: test_14 test_15 test_tip
+VERSIONS = 1.4 1.5 tip
+
+test ::
+		for version in $(VERSIONS);do \
+			echo Running unit tests inside of Go $${version} ...; \
+			docker run --rm -v `pwd`:/go/src/github.com/SUSE/zypper-docker zypper-docker /bin/bash -c "/opt/test.sh -v=$${version}"; \
```
Comment (flavio) [inline Makefile]: You shouldn't have to specify `/bin/bash -c`, by default docker's entrypoint is `sh -c`. Am I missing something?

File: docker/Dockerfile [hunk 2]
```diff
@@ -6,9 +6,17 @@ RUN go get golang.org/x/tools/cmd/vet
 RUN go get github.com/golang/lint/golint
 RUN go get github.com/kisielk/errcheck
 RUN go get github.com/tools/godep
+RUN curl -sL -o /usr/bin/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme
+RUN chmod +x /usr/bin/gimme
+ADD ./test.sh /opt/test.sh
```
Comment (flavio) [inline docker/Dockerfile]: Please use `COPY`, it's better than `ADD` in this context (see [this](https://docs.docker.com/articles/dockerfile_best-practices/#add-or-copy))

File: docker/Dockerfile [hunk 3]
```diff
@@ -6,9 +6,17 @@ RUN go get golang.org/x/tools/cmd/vet
...
 
 WORKDIR /go/src/github.com/SUSE/zypper-docker
 ENV GOPATH=/go:/go/src/github.com/SUSE/zypper-docker/Godeps/_workspace/
+ENV GOROOT_BOOTSTRAP=/opt/go1.4
+ENV GIMME_OS=linux
+ENV GIMME_ARCH=amd64
 
 RUN useradd -m travis
 USER travis
+
```
Comment (flavio) [inline docker/Dockerfile]: Not really important, but this empty line is not needed.

**Comment by flavio** [comment]
On the overall looks good, I'm a bit "scared" by the `for` cycle inside of the Makefile, but there's no other solution AFAIK.

Waiting for @mssola's feedback.

**Comment by mssola** [comment]
LGTM :clap: Just fix the issues raised by @flavio

**Pull request merged**